plugins/lime-plugin-admin/src/adminPage.js

```diff
@@ -35,12 +35,21 @@ const style = {
 };
 
 
-export const Admin = ({ adminLogin, changeConfig, showNotification, selectedHost, nodeData, loading, authStatus }) => {
+export const Admin = ({ changeConfig, showNotification, nodeData, loading, redirect, error }) => {
+	const { nodeHostname, changeNode } = useAppContext();
```

Comment:
it looks like useAppContext became a singleton on the system. would encourage to check if it is essential to be used in that way or a refactor is needed.